lib/google_search_data_viewer_web/templates/layout/app.html.heex

```diff
@@ -1,4 +1,16 @@
 <main class="container">
+  <nav>
+  <ul>
+    <%= if @current_user do %>
+      <li>Signed in as <%= @current_user.email %></li>
+      <li><%= link "Sign out", to: Routes.session_path(@conn, :delete, @current_user), method: :delete %></li>
+    <% else %>
+      <li><%= link "Sign up", to: Routes.user_path(@conn, :new), method: :get %></li> 
+      <li><%= link "Sign in", to: Routes.session_path(@conn, :new), method: :get %></li>
+    <% end %>
+  </ul>
+  </nav>
```

Comment:
Let update the indent properly,

```suggestion
  <nav>
    <ul>
        <%= if @current_user do %>
          <li>Signed in as <%= @current_user.email %></li>
          <li><%= link "Sign out", to: Routes.session_path(@conn, :delete, @current_user), method: :delete %></li>
        <% else %>
          <li><%= link "Sign up", to: Routes.user_path(@conn, :new), method: :get %></li> 
          <li><%= link "Sign in", to: Routes.session_path(@conn, :new), method: :get %></li>
        <% end %>
    </ul>
  </nav>
```

Reply:
Thanks.

Included [here](https://github.com/liamstevens111/google-search-data-viewer/pull/28/commits/4a3529b5c599220bf36e9c9d7eab95d819c66441)